src/Plugin/Replace/Payload.php

```diff
@@ -20,12 +20,11 @@
  * This is simple do nothing request that handle empty queries
  * which can be as a result of only comments in it that we strip
  */
-final class Payload extends BasePayload
-{
+final class Payload extends BasePayload {
 	public string $path;
 
 	public string $table;
-	/** @var array <string, string> */
+	/** @var array<string, bool|float|int|string> */
```

Comment:
Sounds like a good idea to use mixed instead of `bool|float|int|string`, but to be checked